apps/README.md

```diff
@@ -27,7 +27,8 @@ npm run build
 ### Seeing your development version of Apps in Dashboard
 
 1. To make your changes show up in dashboard, do the following after the first time you build apps:
-  - Add `use_my_apps: true` to your locals.yml config file, by editing it yourself or running the following command from the repository root: `echo "use_my_apps: true" >> locals.yml`
+  - Add `use_my_apps: true` to your locals.yml config file.
+    - If you don't have a locals.yml, start by copying locals.yml.default (`cp locals.yml.default locals.yml`)
```

Comment:
Now that "rake install" is doing this automatically, should this documentation be updated?